src/cls/rgw/cls_rgw_ops.h

```diff
@@ -240,7 +240,7 @@ struct rgw_cls_unlink_instance_op {
   std::string olh_tag;
   rgw_zone_set zones_trace;
 
-  rgw_cls_unlink_instance_op() : olh_epoch(0), log_op(false), bilog_flags(0) {}
+  rgw_cls_unlink_instance_op() : olh_epoch(0), log_op(false) {}
```

Comment:
i think this should still initialize bilog_flags to 0

Reply:
done